exspy/misc/eels/tools.py

```diff
@@ -60,16 +60,14 @@ def _estimate_gain(
         variance2fit = variance
         average2fit = average
 
-    fit = np.polyfit(average2fit, variance2fit, pol_order)
+    fit = np.polynomial.Polynomial.fit(average2fit, variance2fit, pol_order)
```

Comment:
The whole function seems to be missing a test as all touched lines raise a warning.